javascript/stimulus_reflex.js

```diff
@@ -397,7 +403,7 @@ const initialize = (application, initializeOptions = {}) => {
     'stimulus-reflex',
     controller || StimulusReflexController
   )
-  debugging = !!debug
+  setDebug(!!debug)
```

Comment:
Geniunely just curious about your thinking and preferences here. My default thought would be to coerce the value in the `setDebug` method, instead of at the various calling sites.

Reply:
Coincidence... didn’t pay enough attention when refactoring 😉 thanks